genomepy/genome.py

```diff
@@ -137,9 +150,40 @@ def _read_metadata(self):
                     accession = p.assembly_accession(genome)
                     metadata["assembly_accession"] = accession
 
-        write_readme(readme, metadata, lines)
+        write_readme(self.readme_file, metadata, lines)
         return metadata
 
+    def get_annotations(self):
+        """returns the file paths to the (gzipped) annotation files"""
+        pattern = os.path.join(self.genome_dir, self.name + ".annotation.")
+        gtf = glob(pattern + "gtf*")
+        bed = glob(pattern + "bed*")
+        return gtf, bed
+
+    def get_gaps_file(self):
```

Comment:
You can change this to be a `@property`

```
@property
def gaps_file(self):
```
and then line 53 can also be removed.
This can also be done for the others.
```